Example/ViewController.swift

```diff
@@ -42,17 +29,15 @@ class ViewController: UIViewController {
 
         player.on(Event.error) { userInfo in print("on Error: \(String(describing: userInfo))") }
 
-        player.on(Event.requestFullscreen) { _ in print("on Enter Fullscreen") }
-
-        player.on(Event.exitFullscreen) { _ in print("on Exit Fullscreen") }
-
         player.on(Event.stalled) { _ in print("on Stalled") }
 
         player.on(Event.requestFullscreen) { _ in
+            self.showAlert(with: "Fullscreen", message: "Entrar em modo fullscreen")
             self.player.setFullscreen(true)
         }
 
         player.on(Event.exitFullscreen) { _ in
+            self.showAlert(with: "Fullscreen", message: "Sair do modo fullscreen")
```

Comment:
I suggest adding a log.
The alert could be removed if it is not  required for testing.